src/app/js/src/reducers.js

```diff
@@ -1,16 +1,82 @@
 import { combineReducers } from 'redux';
 
-import { INC } from './actions';
+import Map from 'ol/map';
+import TileLayer from 'ol/layer/tile';
+
+import {
+    INIT_MAP,
+    CHANGE_LAYER_NAME,
+    CHANGE_LAYER_URL,
+    CHANGE_TILE_JSON,
+    CLEAR_SCREEN,
+    CHANGE_SHARE_LINK,
+} from './actions';
+
+const url = 'https://a.tile.openstreetmap.org/{z}/{x}/{y}.png';
+
+function getDefaultTileJSON() {
+    return [
+        {
+            tilejson: '2.2.0',
+            name: 'base',
+            version: '1.0.0',
+            scheme: 'xyz',
+            tiles: [
+                url,
+            ],
+        },
+    ];
+}
 
 const initialState = {
     n: 0,
+    baseLayer: new TileLayer({}),
+    layers: [],
+    map: new Map({}),
```

Comment:
It's not a good practice to store classes in the Redux store because they can't easily be serialized and redux will have a hard time figuring out if they've changed when updates come through. This applies to the `TileLayer` as well.

In our other apps, we've kept the map instance local to the component that initializes and interacts with it. Any properties that other components need to know about (the zoom level or map center, for instance), can be added to redux directly.

Here's an example map component: https://github.com/azavea/epa-star/blob/develop/src/app/js/src/developer/components/PropertySearchMap.jsx